package/base-files/files/lib/functions/system.sh

```diff
@@ -224,3 +224,15 @@ macaddr_canonicalize() {
 
 	printf "%02x:%02x:%02x:%02x:%02x:%02x" 0x${canon// / 0x} 2>/dev/null
 }
+
+rename_interface_by_mac() {
```

Comment:
Since only octeon is using that for now, I would keep it in the corresponding 03_rename_interfaces until it's required elsewhere.

Reply:
sure, thanks. moved function to `03_rename_interfaces`.